jsinterop-ts-defs-processor/src/main/java/com/vertispan/tsdefs/model/ParameterizedTsType.java

```diff
@@ -49,4 +50,18 @@ private String emitTypeArgs(String parentNamespace) {
             Collectors.joining(
                 ",", optionalAffix("<", typeArguments), optionalAffix(">", typeArguments)));
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+    if (!(o instanceof ParameterizedTsType)) return false;
+    if (!super.equals(o)) return false;
+    ParameterizedTsType that = (ParameterizedTsType) o;
+    return Objects.equals(typeArguments, that.typeArguments);
```

Comment:
note that this can never be null, so just plain typeArguments.equals(that.typeArguments) will work

it might be better to mark that field as final, to be clear that it can never be assigned to

Reply:
Yes right, I will fix this.